File: specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2021-11-01/examples-js/iotconnectors/iotconnector_fhirdestination_Get.js
const { HealthcareApisManagementClient } = require("@azure/arm-healthcareapis");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the properties of the specified Iot Connector FHIR destination.
 *
 * @summary Gets the properties of the specified Iot Connector FHIR destination.
 * x-ms-original-file: specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2021-11-01/examples/iotconnectors/iotconnector_fhirdestination_Get.json
 */
async function getAnIoTConnectorDestination() {
  const subscriptionId = "subid";
  const resourceGroupName = "testRG";
  const workspaceName = "workspace1";
  const iotConnectorName = "blue";
  const fhirDestinationName = "dest1";
  const credential = new DefaultAzureCredential();
  const client = new HealthcareApisManagementClient(credential, subscriptionId);
  const result = await client.iotConnectorFhirDestination.get(
    resourceGroupName,
    workspaceName,
    iotConnectorName,
    fhirDestinationName
  );
  console.log(result);
}

getAnIoTConnectorDestination().catch(console.error);

File: specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2024-03-31/examples-java/iotconnectors/iotconnector_fhirdestination_Get.java

/**
 * Samples for IotConnectorFhirDestination Get.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2024-03-31/examples/iotconnectors/
     * iotconnector_fhirdestination_Get.json
     */
    /**
     * Sample code: Get an IoT Connector destination.
     * 
     * @param manager Entry point to HealthcareApisManager.
     */
    public static void
        getAnIoTConnectorDestination(com.azure.resourcemanager.healthcareapis.HealthcareApisManager manager) {
        manager.iotConnectorFhirDestinations().getWithResponse("testRG", "workspace1", "blue", "dest1",
            com.azure.core.util.Context.NONE);
    }
}
